specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-06-01/managedClusters.json

```diff
@@ -676,6 +676,9 @@
         "x-ms-examples": {
           "Create/Update Agent Pool": {
             "$ref": "./examples/AgentPoolsCreate_Update.json"
```

Comment:
do we need to update the json file name? since we separate create and update scenario now if I understand correctly.

Reply:
these are different examples - one covering Create & Update (using a full object) and another covering a Delta Update, where these optional fields aren't specified

In this example the `count` field is required for a manually scaled cluster (in the Create/Update example) - but cannot be sent for an automatic cluster (in the Update example)

Reply:
There is no difference between using full object and delta update in Updating scenario. Using full object, the "count" and "vmsize" is also not a required value.

Reply:
May be better to remove the Update in the AgentPoolsCreate_Update.json since it does not applies to Update. But it's out of this PR's scope.